docs/index.md

```diff
@@ -58,3 +58,60 @@ another route, based on the changes that the filters made to the
 request.
 
 ![Skipper's request and response processing ](/img/req-and-resp-processing.svg)
+
+#### Routing mechanism
+
+The routing executes the following steps in the typical case:
+
+1. Select the best fitting route by matching the request against the
+   predicates. When no route found, respond with the default status code
+   (the default of the default is 404).
```

Comment:
probably a typo `(the default is 404)`

Reply:
kindof :) i'll try to phrase this in a different way